galileo/src/layer/vector_tile_layer/style.rs

```diff
@@ -286,16 +303,22 @@ impl From<VectorTileLineSymbol> for LinePaint {
 }
 
 /// Symbol for polygon geometries.
-#[derive(Debug, Copy, Clone, Serialize, Deserialize, PartialEq)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct VectorTilePolygonSymbol {
     /// Color of the fill of polygon.
-    pub fill_color: Color,
+    pub fill_color: StyleValue<Color>,
```

Comment:
Other symbol types should also use `StyleValue` instead of raw color and numeric values.